model/riscv_types_kext.sail

```diff
@@ -0,0 +1,355 @@
+/*
+ * This file contains types, mappings and functions used across the
+ * cryptography extension instructions.
+ *
+ * This file must be included in the model build whatever the value of XLEN.
+ */
+
+/* 
+ * Cryptography extension shared / utility functions
+ * ----------------------------------------------------------------------
+ */
+
+/* Auxiliary function for performing GF multiplicaiton */
+val xt2 : bits(8) -> bits(8)
+function xt2(x) = {
+  (x << 1) ^ ( match (bit_to_bool(x[7]) ) {
```

Comment:
Weird spacing here around the parens (which then also is a bit misleading as it looks like the ones with spaces around them are a pair, but they're not, and that wouldn't make sense), also just use if/else instead of matching on a bool.

Reply:
Yeah this was a cursed use of `match`. Changed to `if`.